akiraux/Akira
# Save file format

## Summary / How this PR fixes the problem?
Implement the standard file operations to save, open, and save as, an Akira file.

## Steps to Test
* Create artboards and shapes
* Save the file
* Open the your OS file manager, browse to the newly saved file, view hidden files.
* Enter the `lock` folder and open the `content.json` file to see what was saved.

## Screenshots 
Coming soon...

## This PR fixes/implements the following **bugs/features**:
- [x] Save file `.akira` with all the currently supported objects and properties.
- [x] Allow `Save As...` action to create a copy or override the existing one if selected.
- [x] Open a previously saved file.
- [x] Prevent opening the same file twice.

- Fixes #106
